pkg/api/types.go

```diff
@@ -573,6 +573,9 @@ type MasterProfile struct {
 	FQDN string `json:"fqdn,omitempty"`
 	// True: uses cosmos etcd endpoint instead of installing etcd on masters
 	CosmosEtcd *bool `json:"cosmosEtcd,omitempty"`
+
+	// Enable an alternative SSH port to prevent the default port 22 being blocked
```

Comment:
> to prevent the default port 22 being blocked
this doesn't prevent port 22 from being blocked

```suggestion
	// Enables an alternative SSH port in addition to port 22
```